deploy.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
@mic-smith so you suggest just adding:

`set -e
set -o pipefail`

straight after the the shebang?